.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
+  "rules": {
+    "indent": [
+      2,
+      "tab"
+    ],
+    "quotes": [
+      2,
+      "single"
+    ],
+    "linebreak-style": [
+      2,
+      "unix"
+    ],
+    "semi": [
+      2,
+      "never"
+    ],
+    "comma-dangle": [
+      2,
+      "always-multiline"
+    ],
+    "array-bracket-spacing": [
+      2,
+      "always"
+    ]
+  },
+  "env": {
+    "es6": true,
```

Comment:
Actually, I'll correct myself here: as long as Babel isn't in the pipeline, linting should forbid ES6 features.